cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -257,7 +260,7 @@ func runAgent(ctx context.Context,
 		LogWithTimezone:     "",
```

Comment:
Maybe out of scope for this PR, but do we see a value in allowing customer to configure a max size for the agent log in case they want to reduce disk usage or retain more agent logs? If so, maybe we can opt to use  "file" as `LogTarget` since lumberjack is using [hardcoded config](https://github.com/aws/telegraf/blob/aws-cwa-release-1-23-0/logger/logger.go#L193)

Reply:
I think this is out of scope of this ticket.